nodebuilder/node_test.go

```diff
@@ -45,6 +46,10 @@ func TestLifecycle(t *testing.T) {
 			// ensure the state service is running
 			require.False(t, node.StateServ.IsStopped(ctx))
 
+			// ensure an empty block exists in store
+			err = node.ShareServ.SharesAvailable(ctx, share.EmptyRoot())
+			require.NoError(t, err)
+
```

Comment:
This check is unrelated to the test. There should be a different one

Reply:
A different test for this check you mean? That's fair